index.html

```diff
@@ -447,6 +447,13 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 method</a> [[!BEACON]];</li>
 <li><dfn>"other"</dfn>, if none of the above conditions match.</li>
 </ul>
+<aside data-dfn-for="PerformanceResourceTiming" class="note">
+The "navigation" initiator type includes all types of
+<a data-cite="!FETCH/#navigation-request">navigation requests</a>, including
+<a href=
+    "https://w3c.github.io/ServiceWorker/#ref-for-service-worker-registration-navigation-preload-enabled-flag%E2%91%A2">
```

Comment:
"https://w3c.github.io/ServiceWorker/#service-worker-registration-navigation-preload-enabled-flag" seems to work. What's up with the "%E2%91%A2" bit? 

But also coming back to my earlier question: doesn't this note make more sense in Fetch? We're basically clarifying that "navigation requests" should include navigation preload; seems like that should be spelled out in Fetch. 

/cc @annevk

Reply:
Looking at the fetch spec, from the definition there ('A navigation request is a request whose destination is "document".') it's obvious that navigation preloads are also included, so I'm not sure a note there makes sense.

Let me know what you think, but I'm willing to drop this note entirely, now that we have tests that make sure future implementers get this right.

Reply:
If it's obvious, then I don't think we need it? :)